src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -129,9 +126,8 @@ describe('Grommet', () => {
     /* eslint-enable max-len */
   ].forEach(ua => {
     test(`ssr rendering ${ua.substring(0, 25)}`, () => {
-      const component = renderer.create(<SSRTester ua={ua} />);
-      expect(component.toJSON()).toMatchSnapshot();
-      component.unmount();
+      const {container} = render(<SSRTester ua={ua} />);
+      expect(container.firstChild).toMatchSnapshot();
     });
```

Comment:
I think this test could be improved similar to what was done in the Responsive context tests, right now it is returning small for each option

Reply:
Yeah, I agree. I'll try to improve it when I have some free time.

I even tried to run these tests with them wrapped in an `act()` and it returned "small" for each option as well. I think it is happening something similar to Responsive Context tests.
![Screenshot from 2021-06-20 16-22-07](https://user-images.githubusercontent.com/54550926/122808296-180dec80-d2a3-11eb-9a54-2f7985e97183.png)